static/src/js/containers/DownloadHistoryContainer/DownloadHistoryContainer.jsx

```diff
@@ -1,69 +1,105 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
+import React, {
+  useCallback,
+  useEffect,
+  useState
+} from 'react'
 import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 
-import actions from '../../actions'
 import { DownloadHistory } from '../../components/DownloadHistory/DownloadHistory'
+import RetrievalRequest from '../../util/request/retrievalRequest'
+import { addToast } from '../../util/addToast'
+import { handleError } from '../../actions/errors'
 
 export const mapStateToProps = (state) => ({
-  earthdataEnvironment: state.earthdataEnvironment,
-  retrievalHistory: state.retrievalHistory.history,
-  retrievalHistoryLoading: state.retrievalHistory.isLoading,
-  retrievalHistoryLoaded: state.retrievalHistory.isLoaded
+  authToken: state.authToken,
+  earthdataEnvironment: state.earthdataEnvironment
 })
 
 export const mapDispatchToProps = (dispatch) => ({
-  onFetchRetrievalHistory: (earthdataEnvironment) => {
-    dispatch(actions.fetchRetrievalHistory(earthdataEnvironment))
-  },
-  onDeleteRetrieval:
-    (retrievalId) => dispatch(actions.deleteRetrieval(retrievalId))
+  dispatchHandleError: (errorConfig) => dispatch(handleError(errorConfig))
```

Comment:
`dispatch(actions.handleError())` to follow the pattern of how we dispatch actions